test/reducers/user_reducer_test.js

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'chai'
+
+import userReducer from '../../web/static/js/reducers/user'
+
+
+describe('user reducer', () => {
+  describe('when no state is passed', () => {
+    it('should return the initial state', () => {
+      expect(userReducer(undefined, {})).to.deep.equal([])
+    })
+  })
+
+  describe('when there are no users', () => {
+    it('should handle ADD_USER', () => {
+      const user = { given_name: 'Tiny Rick' }
+      const action = { type: 'ADD_USER', user }
+      const expectedState = [user]
+
+      expect(userReducer([], action)).to.deep.equal(expectedState)
+    })
+  })
+
+  describe('when there are existing users', () => {
+    it('should handle ADD_USER', () => {
+      const user = { given_name: 'Tiny Rick' }
+      const action = { type: 'ADD_USER', user }
+      const expectedState = [{ given_name: 'Morty' }, user]
```

Comment:
would love to avoid the expectation-as-variable in our unit tests. just put it in there inline. 

caviat: if you're going to use the variable in multiple places, it's fine to keep it a variable